plugins/catalog-backend-module-aws/src/awsOrganization/config.ts

```diff
@@ -24,6 +24,11 @@ export type AwsOrganizationProviderConfig = {
    * The role to assume for the processor.
    */
   roleArn?: string;
```

Comment:
may want to add a `@deprecated` on this one's doc comment as well

Reply:
Hey @freben might need more direction on this one. What doc comment needs amended?

Reply:
That should already be there:

https://github.com/backstage/backstage/blob/2b8b4f115a1f3052dc5db9ff14c33b8d0afd8d20/plugins/catalog-backend-module-aws/config.d.ts#L29